admin/client/src/styles/legacy/_forms.scss

```diff
@@ -409,7 +409,7 @@ form.small .field, .field.small {
 				border: none;
 			}
 			&.loading {
-				background: transparent url(../images/network-save.gif) no-repeat $grid-x center;
+				background: transparent url(../images/network-save.gif) no-repeat right $grid-x center;
```

Comment:
@tractorcow 
By changing the `background-position` offset to the 3 value syntax allows to move the ajax spinner background image to the far right, so it does not overlap the CMS action text.

`$grid-x` is 8px;

So `right $grid-x center` means: place the background image **8 pixels from the right**.

The 3rd value is the y-axis, which vertically aligns the image in the middle.

See https://css-tricks.com/almanac/properties/b/background-position/#article-header-id-1